packages/v4/patternfly-docs/content/training/html-css/css-variables-and-overrides.md

```diff
@@ -0,0 +1,227 @@
+
+The custom property name should now be: `--pf-c-alert--m-success`
+
+#### Step 2.3 
+Add the element to the custom property name. The element that is being changed is the title of the alert.
+
+The custom property name should now be: `--pf-c-alert--m-success__title`
+
+#### Step 2.4 
+Add the property that is being modified. In this case, modify the color property of the title in the alert component.
+
+The custom property name should now be: `--pf-c-alert--m-success__title--Color`
+
+#### Step 2.5 
+Define the value of the component level CSS variable. PatternFly's global danger color is: `--pf-global--danger-color--100`. You can reference PatternFly’s <a href="/guidelines/colors/#primary-colors" target="_blank">global colors documentation</a> for more information.
+
+Assign the custom property name `(--pf-c-alert--m-success__title--Color)` that is already inside the `.pf-c-alert{}` block to the global danger color.
+
+It should look like this: `--pf-c-alert--m-success__title--Color: var(--pf-global--danger-color--100);`
+
+The resulting alert should match Figure 1.
+
+**Figure 1**
+<img src="../img/alert-red-title.png" alt="Alert component with a red title" width="868" />
+
+## Part 2: Overriding PatternFly global properties
+In PatternFly, global properties follow this general formula:
+`--pf-global--concept--PropertyCamelCase--modifier--state`
+
+Global properties are prefixed with the word global.
+ - A `concept` is something like a spacer or main-title.
+ - A `PropertyCamelCase` is something like `BackgroundColor` or `FontSize`.
+ - A `modifier` is something like sm, or lg.
+ - A `state` is something like hover, or expanded. They are concepts, never tied to an element or component. This is incorrect: `--pf-global--h1--FontSize`. This is correct: `--pf-global--FontSize--3xl`.
+
+To explore this concept, override the global link color by setting it to the custom property for danger using the provided codesandbox for part 2. The `index.html` file contains a button and expandable component to demonstrate how changing a global variable has an impact across components.
+
+<Button variant="primary" component="a" href="https://codesandbox.io/s/override-global-css-variables-start-llcub8" target="_blank">Codesandbox - Part 2</Button>
+
+### Step 1. Familiarize with `index.html`
+Note of the button component and the expandable section component in the `<body>` of the `index.html` file.
+
+### Step 2. Override a global custom property
+
+#### Step 2.1 
+Write the prefix for the global custom property in the `:root` block in the `styles.css` file. Since global custom properties are prefixed with the word global, use that for the beginning of the custom property.
+
+It should look like this: `--pf-global`
+
+#### Step 2.2 
+Add the concept to the global custom property. The concept for the link color is link.
```

Comment:
```suggestion
Add the concept to the global custom property. The concept for the link color is `link`.
```
Not sure if it'd be better to use backticks or quotation marks in the places where I'm doing so. Mostly want to help with the readability & separate things that are intended to be user-entered values.